clamscan/manager.c

```diff
@@ -1258,6 +1258,25 @@ int scanmanager(const struct optstruct *opts)
 
     if ((opt = optget(opts, "database"))->active) {
         while (opt) {
+            if (optget(opts, "fail-if-cvd-older-than")->enabled) {
+                time_t cvd_age;
+                const long long days = optget(opts, "fail-if-cvd-older-than")->numarg;
+
+                if ((ret = cl_cvdgetage(opt->strarg, &cvd_age)) != CL_SUCCESS) {
+                    logg(LOGG_ERROR, "%s\n", cl_strerror(ret));
+
+                    ret = 2;
```

Comment:
I imagine you don't need to override `ret` here.

Reply:
Please see previous comment on overriding `ret`.

Reply:
> I guess nevermind on this as per my reply to the previous comment. Sorry!

No problem at all, such is collaborating on software! :) I'll get rid of `CL_EOUTDATED_CVD` in that case (and just use the old generic `CL_ECVD`).

I guess nevermind on this as per my reply to the previous comment. Sorry!